### PR TITLE
Help finance view controller

### PR DESCRIPTION
**Pull request by kobsonaut**

### CODE REVIEW

File: TOZ_iOS/GalleryViewController.swift [hunk 1]
```diff
@@ -10,36 +10,37 @@ import Foundation
...
-        if let cell = cell as? GalleryTableViewCell {
+        if let cell = tableView.dequeueReusableCell(withIdentifier: "galleryTableViewCell") as? GalleryTableViewCell {
             let animalInfo = animalsArray[indexPath.row]
             cell.animalName.text = animalInfo.name
             cell.animalType.text = animalInfo.type
             cell.animalImage.image = animalInfo.image
+            return cell
         }
-        return cell
+        return UITableViewCell()
```
Comment (wobrak) [inline TOZ_iOS/GalleryViewController.swift]: What is the purpose to return new object UITableViewCell()?

File: TOZ_iOS/GalleryViewController.swift [hunk 2]
```diff
@@ -10,36 +10,37 @@ import Foundation
...
-
-    var animalsArray = [GalleryEntity]()
-
+    @IBOutlet weak var temporaryName: UILabel!
     @IBOutlet weak var imageView: UIImageView!
+    @IBAction func unwindToGallery(segue: UIStoryboardSegue) {
+    }
     //@IBOutlet weak var indicator: UIActivityIndicatorView!
-
+    var animalsArray = [GalleryEntity]()
```
Comment (Gorczyk89) [inline TOZ_iOS/GalleryViewController.swift]: It's better to change GalleryEntity for GalleryItem (check network items catalog).

File: TOZ_iOS/GalleryViewController.swift [hunk 3]
```diff
@@ -10,36 +10,37 @@ import Foundation
...
         return animalsArray.count
     }
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: "galleryTableViewCell", for: indexPath)
-        if let cell = cell as? GalleryTableViewCell {
+        if let cell = tableView.dequeueReusableCell(withIdentifier: "galleryTableViewCell") as? GalleryTableViewCell {
             let animalInfo = animalsArray[indexPath.row]
             cell.animalName.text = animalInfo.name
             cell.animalType.text = animalInfo.type
             cell.animalImage.image = animalInfo.image
```
Comment (Gorczyk89) [inline TOZ_iOS/GalleryViewController.swift]: Did you think about making model for GalleryTableViewCell?

File: TOZ_iOS/GalleryViewController.swift (hunk 2 above)
Comment (kobsonaut) [inline TOZ_iOS/GalleryViewController.swift]: I know, but at the time I was doing that gallery wasn't connected with BE.

File: TOZ_iOS/GalleryDetailViewController.swift [hunk 4]
```diff
@@ -9,5 +9,13 @@
 import UIKit
 
 class GalleryDetailViewController: UIViewController {
-
+    override func viewDidLoad() {
```
Comment (ArkadiuszBanas) [inline TOZ_iOS/GalleryDetailViewController.swift]: We don't need it.

File: TOZ_iOS/GalleryViewController.swift [hunk 5]
```diff
@@ -10,36 +10,37 @@ import Foundation
...
         let thirdAnimal = GalleryEntity(name: "Lola", type: "Owczarek", image: UIImage(named: "dog3"))
         animalsArray = [firstAnimal, secondAnimal, thirdAnimal]
+//        temporaryName.text = "TOZ Szczecin"
     }
     // UITableViewDataSource
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         return animalsArray.count
     }
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: "galleryTableViewCell", for: indexPath)
```
Comment (ArkadiuszBanas) [inline TOZ_iOS/GalleryViewController.swift]: Please revert this changes.
